scripts/prepare_sector_network.py

```diff
@@ -2722,8 +2722,11 @@ def add_biomass(n, costs):
             bus2="co2 atmosphere",
             carrier="unsustainable bioliquids",
             efficiency=1,
-            efficiency2=-costs.at["solid biomass", "CO2 intensity"]
-            + costs.at["BtL", "CO2 stored"],
+            efficiency2=(
```

Comment:
in would prefer this to be `-costs.at["oil", "CO2 intensity"]`

Reply:
Done